pkg/test/runner/runner.go

```diff
@@ -45,10 +45,14 @@ func NewRunner(testCase TestCase, c string) (*Runner, error) {
 
 // Run runs the test.
 func (r *Runner) Run() error {
-	if r.cmd == CommandFn {
+	switch r.cmd {
+	case CommandFnEval:
 		return r.runFn()
+	case CommandFnRender:
+		return r.runPipeline()
```

Comment:
Maybe these method names should change. Before they were calling `kpt fn run` and `kpt pipeline run` and now the names here don't make sense. E.g. instead of `r.runFn` and `r.runPipeline` they can be `r.fnRun` and `r.fnRender`

Reply:
Good point. Done.